tests/open_test.py

```diff
@@ -56,5 +56,11 @@ def test_open():
     os.remove(h51)
     os.remove(h52)
 
-
-
+    # be nice to users when converting from unsupported format
+    with tempfile.NamedTemporaryFile(suffix='json') as in_f:
+        in_f.write(b'[]')
+        in_f.flush()
+        with pytest.raises(OSError) as exc:
```

Comment:
On Python3 these are the same, but not Python 2. If you replace it by IOError, it should pass

Reply:
done